mlops/README.md

```diff
@@ -0,0 +1,14 @@
+# MLOps
+
+This directory will contain samples and examples to show how to leverage ART for MLOps, in Machine Learning Platforms.
```

Comment:
Machine Learning Platforms and Pipelines

Reply:
done